docs/hooks/intro.md

```diff
@@ -0,0 +1,132 @@
+# Hooks
+
+`react-plot` provides hook that allows to add functionalities to the plot.
+
+To benefit of those features the hooks have to be wrapped in a `PlotController`.
+
+```jsx live
+function Demo() {
+  return (
+    <PlotController>
+      <Zoomable />
+    </PlotController>
+  );
+
+  function Zoomable() {
+    const zoom = useRectangularZoom();
+    return (
+      <Plot width={300} height={300}>
+        <LineSeries
+          data={[
+            { x: 1, y: 1 },
+            { x: 2, y: 2 },
+            { x: 3, y: 3 },
+            { x: 4, y: 2 },
+            { x: 5, y: 1 },
+          ]}
+        />
+        <Annotations>{zoom.annotations}</Annotations>
+      </Plot>
+    );
+  }
+}
+```
+
+```jsx live
```

Comment:
I'm trying to have a simple example in which we can display the tracking events in a DIV.

However when I use the following code as soon as I mouse down there is a refresh of the component and therefore the mouse move is lost.

What is the simple way to solve this problem ?

Reply:
Where is this page supposed to be? I cannot find it in the deploy preview